README.md

```diff
@@ -308,3 +308,21 @@ The following is equivalent:
 
   See more on [this blog post](https://medium.com/@tomchentw/imagemin-lint-staged-in-place-minify-the-images-before-adding-to-the-git-repo-5acda0b4c57e) for benefits of this approach.
 </details>
+
+### :warning: Important note for users of JetBrains IDEs *(WebStorm, PyCharm, IntelliJ IDEA, RubyMine, etc.)*
+
+The JetBrains suite of IDEs [currently has a bug](https://youtrack.jetbrains.com/issue/IDEA-135454) that results in the IDE showing inconsistent changed state for those files changed by any of your `precommit` hooks. A `git diff` will show the results of your `precommit` script, even though using the IDE to compare your local file to the same branch shows that they are identical. This is due to [the IDE's use of the `--only` flag](https://youtrack.jetbrains.com/issue/IDEA-135454#comment=27-2352586) when committing in the UI. 
```

Comment:
I'd rewrite this paragraph and leave only important bits and link to the GH and YouTrack issues.

```
When using IDE's GUI to commit changes with the recommit hook, you might see inconsistent changes in the IDE and command line. This is [known issue] of IDE so if you want this fixed, please vote for it on YouTrack.
```

Please feel free to re-write this since it's just a direction. I don't think all the technical details are that important and they can be acquired in the GH issue. The goal of this note is to provide a quick solution for people who is experiencing this. We should also link to the section in the GH issue itself.